src/dmd/astbase.d

```diff
@@ -1213,9 +1213,10 @@ struct ASTBase
     {
         Expressions* atts;
 
-        extern (D) this(Expressions* atts, Dsymbols* decl)
+        extern (D) this(Loc loc, Expressions* atts, Dsymbols* decl)
```

Comment:
The rest of the codebase uses `cont ref Loc` because `Loc` is big. Ditto throughout.

Reply:
Hmm, perhaps I haven't updated my branch since then. I'll update.

Reply:
Fixed.